pyproject.toml

```diff
@@ -29,7 +29,6 @@ dependencies = [
     "bytecode>=0.15.0; python_version>='3.12'",
     "bytecode>=0.14.0; python_version~='3.11.0'",
     "bytecode>=0.13.0; python_version<'3.11'",
-    "cattrs",
```

Comment:
Generally speaking, what triggered this change? Did we recently run into issues with customer code, or has this just always been an issue so we're fixing it now?